tests/testutils.py

```diff
@@ -182,3 +185,24 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return decorator
+
+
+class PlaywrightTestCase(StaticLiveServerTestCase):
```

Comment:
Using Playwright in tests requires some setup, so I abstracted that away into `PlaywrightTestCase`